airtable.js

```diff
@@ -0,0 +1,52 @@
+// Don't forget to add your API key to the key variable
+const key = ""
```

Comment:
// hard code 
Use environment variables for sensitive information: Instead of hardcoding the API key directly in the code, it's better to use environment variables to store sensitive information like API keys. This helps in keeping the code more secure and makes it easier to manage different environments (e.g., development, production) with different keys.